### ISSUE TITLE
Application name not updated correctly with ${directory}

### ISSUE THREAD

**Issue by souzamatheusp** [gnunn1/tilix#1240]
When setting the application title value to `${appName}: ${directory}`, I found that changing the working directory does not immediately change the window's name.

Steps to reproduce the issue:
1. Set the above value to the application title in the settings
2. Restart Tilix
3. `cd` into some folder

The window title does not update, unless some event outside the program itself causes it to (for instance, unfocusing the window and coming back) or I `cd` into another folder. In the latter case, the title updates to the previous folder I had `cd`'d into, not to the last one.
In short, `${directory}` is always one step behind. Can you reproduce it?

Note (just to eliminate the variable): the presence of other `${keywords}` doesn't seem to affect the issue.

**Comment by gnunn1** [comment]
It works fine for me, as a guess you probably don't have the VTE configured correctly in terms of sourcing ```vte.sh```. See the following for additional information:

https://gnunn1.github.io/tilix-web/manual/vteconfig/

**Comment by souzamatheusp** [comment]
I had done those things. It's ok now, though. Might have been a logout-login thing. Sorry for not checking that before. Thanks for the help, and very nice app you have there!
I'm closing the issue.
